raiden-cli/src/types.ts

```diff
@@ -0,0 +1,34 @@
+import { Raiden } from 'raiden-ts';
```

Comment:
Nitpick: could you, please, on this module, import like:
```suggestion
import type { Raiden } from 'raiden-ts';
```
This is TypeScript 3.9 way of telling this import will actually be removed from transpiled output, as these imports are used only for typing, and this is an interface/types-only file.
We still need to apply this concept to our codebase, so feel free to ignore (as all nitpicks), but is more as a headsup of this nice recent language addition.